strapi/strapi
# Use server timezone for logging

### What does it do?
When `STRAPI_LOG_SERVER_TIME` is true, use native `TZ` env variable to determine Locale server timezone and format the date with `STRAPI_LOG_DATETIME_FORMAT` (default value yyyy-MM-dd'T'HH:mm:ss.sss)

### Why is it needed?

Logging with server time

### How to test it?

On the .env file : 
* activate the feature buy putting `STRAPI_LOG_SERVER_TIME=true`
* change the timezone to another timezone rather that UTC. (Ex. `TZ=Asia/Shanghai`) 
Launch your Strapi application and check if the logs date time are correct with the timezone you chose.
